File: 2418-sort-the-people/2418-sort-the-people.java
class Solution {
    public String[] sortPeople(String[] names, int[] heights) {
        for(int i = 0;i<heights.length;i++){
            for(int j = 0;j<heights.length;j++){
                if(heights[i]>heights[j]){//Ascending Order
                    //heights swap
                    int tempH = heights[i];
                    heights[i] = heights[j];
                    heights[j] = tempH;
                   // names swap
                   String tempN = names[i];
                   names[i]=names[j];
                   names[j] = tempN; 
                }
            }
        }
        return names;
    }
}

File: 2418-sort-the-people/2418-sort-the-people.py
class Solution:
    """using python"""
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        result = []
        #loop through all people
        for _ in range(len(heights)):
            max_height = max(heights)#find tallest heights
            index = heights.index(max_height)#get index of tallest

            result.append(names[index])#add corresponding names
            heights.pop(index)#remove used height
            names.pop(index)#remove used name
        return result